samples/csharp/getting-started/DatabaseIntegration/DatabaseIntegration/Program.cs

```diff
@@ -0,0 +1,102 @@
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Categorical;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+
+namespace DatabaseIntegration
+{
+    public class Program
+    {
+        private static string Download(string baseGitPath, string dataFile)
+        {
+            using (WebClient client = new WebClient())
+            {
+                client.DownloadFile(new Uri($"{baseGitPath}"), dataFile);
+            }
+
+            return dataFile;
+        }
+
+        public static string DownloadAdultDataset()
+            => Download("https://raw.githubusercontent.com/dotnet/machinelearning/244a8c2ac832657af282aa312d568211698790aa/test/data/adult.train", "adult.txt");
+
+        public static List<AdultCensus> GetAdultData()
+        {
+            string adultDataset = DownloadAdultDataset();
+            string dataLine;
+            var dataset = new List<AdultCensus>();
+            using (StreamReader reader = new StreamReader(adultDataset))
+            {
+                // read the header
+                dataLine = reader.ReadLine();
+                while ((dataLine = reader.ReadLine()) != null)
```

Comment:
I prefer how @jwood803 accomplished this using LINQ in https://github.com/dotnet/machinelearning/pull/2902/files#diff-8696be27c85dee937c625ba6c27d564fR55, but I would start with ReadLines instead of ReadAllLines.